newsletter-323: translate into Chinese (#132)
PrimitivesLane/bitcoinops.github.io: _posts/zh/newsletters/2024-10-04-newsletter.md

- [Bitcoin Core #30043][] 引入了[端口控制协议(PCP)][rfcpcp]的内置实现，以支持 IPv6 pinholing，允许节点在不需要手动配置路由器的情况下变得可访问。此更新用 PCP 替换了现有的用于 IPv4 端口映射的 `libnatpmp` 依赖，同时还实现了一个回退机制到 [NAT 端口映射协议(NAT-PMP)][rfcnatpmp]。虽然 PCP / NAT-PMP 功能默认是禁用的，但在未来的版本中可能会改变。见周报[#131][news131 natpmp]。

- [Bitcoin Core #30510][] 为 `Mining` 接口添加了一个进程间通信(IPC)包装器 (见周报[#310][news310 stratumv2])，允许独立的 [Stratum v2][topic pooled mining] 挖矿进程通过连接和控制 `bitcoin-node` 进程来创建、管理和提交区块模板(见周报[#320][news320 stratumv2])。[Bitcoin Core #30409][]扩展了 `Mining` 接口，增加了一个新的 `waitTipChanged()` 方法，该方法可以检测新区块的到来，然后将新的区块模板推送给已连接的客户端。`waitfornewblock`、`waitforblock` 和 `waitforblockheight` RPC 方法已被重构以使用它。

- [Core Lightning #7644][] 为 `hsmtool` 工具添加了 `nodeid`命令，该命令返回给定的 `hsm_secret` 备份文件的节点标识符，以将备份与其特定节点匹配，并避免与其他节点混淆。

- [Eclair #2848][] 实现了可扩展的[流动性广告][topic liquidity advertisements]，如 [BOLTs #1153][]所指出的那样，允许卖家在其 `node_announcement` 消息中广告他们愿意向买家出售流动性的费率，然后买家可以连接并请求流动性。这可以在创建[双向注资通道][topic dual funding]时使用，或在通过[通道拼接][topic splicing]向现有通道添加额外流动性时使用。

- [Eclair #2860][] 添加了一个可选的 `recommended_feerates` 消息，供节点向其对等节点告知他们希望用于注资通道交易的可接受费率。如果一个节点拒绝了对等节点的注资请求，对等节点将理解这是由于费率原因。

> - [Eclair #2861][] 实现了 on-the-fly 注资，如 [BLIPs #36][]所指出的那样，允许没有足够入账流动性的客户端通过[流动性广告][topic liquidity advertisements]协议(见上面 PR)向对等节点请求额外流动性，以接收付款。流动性卖家承担[双向注资通道][topic dual funding]或是[通道拼接][topic splicing]交易的链上交易费用，但在付款路由费时由买家支付。如果金额不足以支付交易确认所需的链上费用，卖家可以双花它以在其他地方使用他们的流动性。

Reviewer: 考虑翻成 “空中加油注资”

> - [Eclair #2861][] 实现了 on-the-fly 注资，如 [BLIPs #36][]所指出的那样，允许没有足够入账流动性的客户端通过[流动性广告][topic liquidity advertisements]协议(见上面 PR)向对等节点请求额外流动性，以接收付款。流动性卖家承担[双向注资通道][topic dual funding]或是[通道拼接][topic splicing]交易的链上交易费用，但在付款路由费时由买家支付。如果金额不足以支付交易确认所需的链上费用，卖家可以双花它以在其他地方使用他们的流动性。

Reviewer: ```suggestion
- [Eclair #2861][] 实现了 on-the-fly 注资，如 [BLIPs #36][]所指出的那样，允许没有足够入账流动性的客户端通过[流动性广告][topic liquidity advertisements]协议(见上面 PR)向对等节点请求额外流动性，以接收付款。流动性卖家承担[双向注资通道][topic dual funding]或是[通道拼接][topic splicing]交易的链上交易费用，但后续便可在路由支付时得到买方的支付。如果金额不足以支付交易确认所需的链上费用，卖家可以双花它以在其他地方使用他们的流动性。
```

> - [Eclair #2875][] 实现了注资费用信用，如 [BLIPs #41][] 所指出的那样，允许 on-the-fly 注资 (见上面 PR) 客户端接受太小而无法支付通道链上费用的付款。一旦积累了足够的费用信用，就会使用费用信用创建一个链上交易，如通道注资或[通道拼接][topic splicing]。客户端依赖流动性提供者在未来的交易中兑现费用信用。

Reviewer: ```suggestion
- [Eclair #2875][] 实现了注资费用积分，如 [BLIPs #41][] 所指出的那样，允许 on-the-fly 注资 (见上面 PR) 客户端接受太小而不足以支付通道链上费用的付款（但同时积累费用积分）。一旦积累了足够的费用积分，对手就会创建一个链上交易，如通道注资或[通道拼接][topic splicing]。客户端要依赖流动性提供者在未来的交易中兑现费用积分。
```

> - [LDK #3303][] 为入账支付添加了一个新的 `PaymentId`，以改善幂等事件处理。这允许用户在节点重启期间重放事件时轻松检查事件是否已被处理。依赖 `PaymentHash` 时可能出现的重复处理风险被消除。The `PaymentId` 是支付中包含的 [HTLC][topic htlc] 通道标识符和 HTLC 标识符对的基于哈希的消息认证码（HMAC）。

Reviewer: ```suggestion
- [LDK #3303][] 为入账支付添加了一个新的 `PaymentId`，以改善幂等事件处理。这允许用户在节点重启期间重放事件时轻松检查事件是否已被处理。依赖 `PaymentHash` 时可能出现的重复处理风险被消除。`PaymentId` 是支付中包含的 [HTLC][topic htlc] 通道标识符和 HTLC 标识符对的基于哈希的消息认证码（HMAC）。
```
